homeassistant/components/sma/sensor.py

```diff
@@ -127,7 +127,11 @@ def _check_sensor_schema(conf):
             backoff -= 1
             return
 
-        values = await sma.read(used_sensors)
+        try:
+            values = await sma.read(used_sensors)
+        except TypeError:
```

Comment:
Shouldn't a general error like this be caught in the library? Libraries should preferably only raise their own exceptions when needed. How should the user of the library know to catch this error?

Reply:
was probably lazy :-)